cli/commands/console.go

```diff
@@ -166,17 +168,31 @@ func (o *consoleOptions) run() error {
 	consoleURL := fmt.Sprintf("http://%s:%s/", o.Address, o.ConsolePort)
 
 	if !o.DontOpenBrowser {
-		o.EC.Spin(color.CyanString("Opening console using default browser..."))
-		defer o.EC.Spinner.Stop()
+		if o.Browser != "" {
+			o.EC.Spin((color.CyanString("Opening console on: %s", o.Browser)))
+			defer o.EC.Spinner.Stop()
+			err = open.RunWith(consoleURL, o.Browser)
+			if err != nil {
+				o.EC.Logger.WithError(err).Warn("Error opening browser, try to open the url with another browser")
+				return err
+			}
+		} else {
+			o.EC.Spin(color.CyanString("Opening console using default browser..."))
+			defer o.EC.Spinner.Stop()
 
-		err = open.Run(consoleURL)
-		if err != nil {
-			o.EC.Logger.WithError(err).Warn("Error opening browser, try to open the url manually?")
+			err = open.Run(consoleURL)
+			if err != nil {
+				o.EC.Logger.WithError(err).Warn("Error opening browser, try to open the url manually?")
+			}
 		}
 	}
 
 	o.EC.Spinner.Stop()
-	log.Infof("console running at: %s", consoleURL)
+	if o.Browser != "" {
+		log.Infof("console running at: %s on %s", consoleURL, o.Browser)
+	} else {
+		log.Infof("console running at: %s", consoleURL)
+	}
```

Comment:
We don't need to do this here since console is running at this point irrespective of the browser flag.

Change it to just
```go
log.Infof("console running at: %s", consoleURL)
```